### ISSUE TITLE
jx create cluster eks with --domain switch fails to create good ingress

### ISSUE THREAD

**Issue by chuckwilbur** [jenkins-x/jx#1498]
### Summary 
I can successfully stand up an EKS cluster with `jx create cluster eks`, providing my custom dns name when prompted.
However, if instead I run `jx create cluster eks --domain=<my custom dns name>`, the cluster that stands up does not expose Jenkins to the world properly and the installation fails at the "Getting Jenkins API Token" stage.

### Steps to reproduce the behavior
`jx create cluster eks --domain=<my custom dns name>`
when prompted `A local Jenkins X cloud environments repository already exists, recreate with latest? ` answer Yes

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 1.3.185
jenkins x platform 0.0.2191
kubernetes cluster v1.10.3
kubectl            v1.11.2
helm client        v2.10.0+g9ad53aa
helm server        v2.10.0+g9ad53aa
git                git version 2.10.1 (Apple Git-78)
```

### Kubernetes cluster
EKS

### Operating system / Environment
MacOS

### Expected behavior
```
Getting Jenkins API Token
using url http://jenkins.jx.<my custom dns name>/me/configure
logging in
Getting the API Token...
Found API Token
```

### Actual behavior
```
Getting Jenkins API Token
using url http://jenkins.jx.<my custom dns name>/me/configure
unable to automatically find API token with chromedp using URL http://jenkins.jx.<my custom dns name>/me/configure
Please go to http://jenkins.jx.<my custom dns name>/me/configure and click Show API Token to get your API Token
Then COPY the token and enter in into the form below:

? API Token: 
```
`http://jenkins.jx.<my custom dns name>/me/configure` times out in a browser.

The issue appears to lie in the setup of the load balancer and/or the ingress. In the nominal/working case where no `--domain` parameter is passed, one can use `curl <ELB id>.elb.us-east-1.amazonaws.com --header 'Host: jenkins.jx.<my custom dns name>'` and get a valid Jenkins page returned. A similar curl command for the ELB in the error case returns
```
curl: (52) Empty reply from server
```

We have pored over the code, diffs of kubectl output from a working and non-working cluster, and the resources of the clusters in the AWS console and can find no difference to explain why this fails in one case and not the other.

An extremely weird factor is that the answer to the `A local Jenkins X cloud environments repository already exists, recreate with latest?` prompt matters. If you do NOT recreate the local cloud environment the load balancer is created correctly and everything works.

Unfortunately, the `--domain` switch plus recreating the local cloud env is the "run silently in batch mode" case, which I would really like to see work.

**Comment by fabiodouek** [comment]
Hi @chuckwilbur , that's interesting that you are able to create the cluster without custom domains. Myself and others are having issues in creating a vanilla EKS cluster with jx on top. I've tried also 1.3.172, but no luck. https://github.com/jenkins-x/jx/issues/1472

**Comment by scblur869** [comment]
I have the same problem with the ELB not being created correctly as well when trying to install onto AWS in an existing kops created cluster. I have tried to bypass the "recreate" but the elb still never gets created. also on the documentation it says use `` kubectl get service -n kube-system jxing-nginx-ingress-controller  -oyaml | grep hostname `` but this just returns nothing. Actually i have never seen the "hostname" property in a service spec...

**Comment by chuckwilbur** [comment]
My particular issue seems to be fixed as of jx version 1.3.287
